encoder.go

```diff
@@ -38,7 +38,7 @@ func encodeTriggerBody(channels []string, event string, data interface{}, socket
 		payloadData = string(dataBytes)
 	}
 	if len(payloadData) > maxEventPayloadSize {
-		return nil, errors.New("Data must be smaller than 10kb")
+		return nil, errors.New("Event payload exceeded maximum size")
```

Comment:
How do you see us reporting also the `len(payloadData)` in the error? Might be useful to debug things.

Reply:
Good idea, added it in 952fddb.